src/js/actions/search/libraries.js

```diff
@@ -91,7 +91,7 @@ define(function (require, exports) {
                         name: title,
                         pathInfo: library.name + ": " + strings.SEARCH.CATEGORIES[categoryKey],
                         iconID: _getSVGClass([categoryKey]),
-                        category: ["LIBRARY", library.name, categoryKey]
+                        category: ["LIBRARY", library.id.replace(/-/g, "."), categoryKey]
```

Comment:
OK, this is a longer term architectural problem which we should figure out how to address in the future. This seems good for now though. Thanks!